osu.Game/Storyboards/StoryboardSprite.cs

```diff
@@ -33,10 +33,8 @@ public double StartTime
 
                 foreach (var l in loops)
                 {
-                    if (!(l.EarliestDisplayedTime is double lEarliest))
-                        continue;
-
-                    earliestStartTime = Math.Min(earliestStartTime, lEarliest);
+                    if (l.EarliestDisplayedTime is double loopEarliestDisplayTime)
+                        earliestStartTime = Math.Min(earliestStartTime, l.LoopStartTime + loopEarliestDisplayTime);
```

Comment:
I've changed this line as discussed on voice, due to the previous commit version reading a bit weirdly.